ibis/backends/tests/test_aggregation.py

```diff
@@ -80,21 +85,24 @@ def mean_udf(s):
         lambda t: (t.int_col % 3).mode(),
         lambda t: (t.int_col % 3).mode().iloc[0],
         id="mode",
-        marks=pytest.mark.notyet(
-            [
-                "bigquery",
-                "clickhouse",
-                "datafusion",
-                "impala",
-                "mysql",
-                "mssql",
-                "pyspark",
-                "trino",
-                "druid",
-                "oracle",
-            ],
-            raises=com.OperationNotDefinedError,
-        ),
+        marks=[
+            pytest.mark.notyet(
+                [
+                    "bigquery",
+                    "clickhouse",
+                    "datafusion",
+                    "impala",
+                    "mysql",
+                    "mssql",
+                    "pyspark",
+                    "trino",
+                    "druid",
+                    "oracle",
+                ],
+                raises=com.OperationNotDefinedError,
+            ),
+            pytest.mark.broken(["flink"], "WIP", raises=com.OperationNotDefinedError),
```

Comment:
We try to reserve using `broken` for things that are both truly broken and also unrelated to the PR in question (maybe some upstream thing in a different backend breaks while you are adding functionality for a separate backend).

All of these that are marked as WIP should be changed over to either `notimpl` or `notyet`, depending on whether it's planned work or waiting on upstream improvements.

Reply:
@gforsyth All of the "WIP" are from me, just as a way to mark things that I didn't even look into/just marked as failing (as suggested in the sync call), in order to get a fully-green test suite. I can switch them all to `notimpl`, as you recommend.

The first thing we are doing is going through the "WIP" ones, to actually evaluate what needs to be done on them.

Reply:
Done!